projects/plugins/jetpack/modules/search/instant-search/components/search-results.jsx

```diff
@@ -73,6 +77,19 @@ class SearchResults extends Component {
 				num,
 				corrected_query
 			);
+		} else if ( isMultiSite ) {
+			const group = getAvailableStaticFilters().filter( item => item.filter_id === 'group_id' );
+			const allP2 =
+				group.length === 1 && group[ 0 ].values
+					? group[ 0 ].values.filter( item => item.value !== MULTISITE_NO_GROUP_VALUE )
+					: {};
+			const p2Name = allP2[ 0 ]?.name ? allP2[ 0 ].name : __( 'All P2', 'jetpack' );
+			return sprintf(
+				/* translators: %1$s: number of results. - %2$s: site name. */
+				_n( 'Found %1$s result in %2$s', 'Found %1$s results in %2$s', total, 'jetpack' ),
+				num,
+				p2Name
+			);
```

Comment:
This causes non-P2 sites to show `Found <number of results> result in All P2`. I recommend two changes here:

1. We need to ensure that this P2-specific title is only shown for eligible P2 sites; the `isMultiSite` boolean is not working as expected. 
2. Are we using `_n` here correctly? I sometimes see the non-pluralized title for multiple results.

Here's a screenshot that demonstrates both issues on my non-P2 test site:
> <img width="254" alt="Screen Shot 2021-06-22 at 2 11 17 PM" src="https://user-images.githubusercontent.com/4044428/122992753-b792b400-d363-11eb-938e-09e928574729.png">

Reply:
> We need to ensure that this P2-specific title is only shown for eligible P2 sites; the isMultiSite boolean is not working as expected.

Good catch, fixed.

> Are we using _n here correctly? I sometimes see the non-pluralized title for multiple results.

Looks good to me, did not manage to reproduce the above scenario, will take another closer look at it